service/error_handlers.py

```diff
@@ -0,0 +1,84 @@
+# TODO: Copyright header
```

Comment:
We were missing error handlers, so I added them in as well. I added a few basic tests, I think we will eventually need a test for a `500` response.